### PR TITLE
Build assets on release

### PR DESCRIPTION
**Pull request by yguedidi**
Assuming the release script is used only through the GitHub action

similar to https://github.com/wallabag/docker/pull/362 but for ZIP releases

### CODE REVIEW

File: .github/workflows/upload-release-package.yml [hunk 1]
```diff
@@ -29,6 +29,12 @@ jobs:
         env:
           COMPOSER_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
+      - name: "Install Node"
+        uses: actions/setup-node@v3
+        with:
+          node-version-file: ".nvmrc"
+          cache: "yarn"
```
Comment (yguedidi) [inline .github/workflows/upload-release-package.yml:36]: copied from the Assets workflow haha

File: scripts/release.sh [hunk 2]
```diff
@@ -10,6 +10,8 @@ ENV=$4
 rm -rf "${TMP_FOLDER:?}"/"$RELEASE_FOLDER"
 mkdir "$TMP_FOLDER"/"$RELEASE_FOLDER"
 git clone https://github.com/wallabag/wallabag.git --single-branch --depth 1 --branch $1 "$TMP_FOLDER"/"$RELEASE_FOLDER"/"$VERSION"
+cd "$TMP_FOLDER"/"$RELEASE_FOLDER"/"$VERSION" && yarn install --non-interactive
+cd "$TMP_FOLDER"/"$RELEASE_FOLDER"/"$VERSION" && yarn run --non-interactive build:prod
```
Comment (yguedidi) [inline scripts/release.sh:14]: tested those commands locally and they work

**Review by j0k3r — COMMENTED** [review]
Shouldn't you remove the assets from git then? And maybe also remove the `assets` workflow?

**Comment by yguedidi** [comment]
> Shouldn't you remove the assets from git then? And maybe also remove the assets workflow?

@j0k3r yes! but wanted to have this merged first and see it working in the next release before cleaning the code :)
I can of course clean now if you want!

**Comment by j0k3r** [comment]
I think you can clean it right now :)

**Comment by yguedidi** [comment]
@j0k3r answering you I realize that with assets still in the repo the release will work anyway... 😅 so I'll clean now

**Review by j0k3r — APPROVED** [review]

**Pull request merged**